static/scss/common.scss

```diff
@@ -63,6 +63,17 @@ a.link-button {
   font-size: 20px;
 }
 
+.btn-denim-blue {
+  background-color: $denim-blue !important;
+  border-color: $denim-blue !important;
```

Comment:
why we need !important? won't they work without it ?

Reply:
Removed